src/components/PeopleFilters/CenturyFilter.tsx

```diff
@@ -0,0 +1,52 @@
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { SearchLink } from '../SearchLink';
+
+export const CenturyFilter:React.FC = () => {
+  const ages = ['16', '17', '18', '19', '20'];
```

Comment:
Consider using a more descriptive variable name instead of `ages` for the centuries array, for example, `centuryList`. This will improve code readability.